main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"net/http"
+	"time"
+
+	"URezL/api"
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	router := mux.NewRouter()
+	router.HandleFunc("/links/cut", api.PostLinkCut).Methods(http.MethodPost)
+	router.HandleFunc("/links/custom", api.PostCustomLink).Methods(http.MethodPost)
+	router.HandleFunc("/login", api.PostLogin).Methods(http.MethodPost)
+	router.HandleFunc("/register", api.PostRegister).Methods(http.MethodPost)
+	router.HandleFunc("/links", api.GetLinks).Methods(http.MethodGet)
+	router.HandleFunc("/links", api.DeleteLink).Methods(http.MethodDelete)
```

Comment:
Sure, it is a legal way to use a delete method. But it doesn't fit in a Rest style. We are deleting resource. So, we must provide resource via URL. So, what about links identification?